### PR TITLE
follow-up changes from discussion in issue #220

### PR DESCRIPTION
**Pull request by wwang0**
Two commits here to make doc clearer:
1. treat vrgather scalar register index as unsigned
2. Prelace imm with uimm for shift instructions

### CODE REVIEW

**Comment by aswaterman** [comment]
The first issue was already addressed by https://github.com/riscv/riscv-v-spec/pull/222 - can you resubmit a PR that only addresses the shift immediate name?

**Comment by wwang0** [comment]
@aswaterman , my first patch is the follow up of #222, it tell explicitly that the scalar register index should also be unsigned, I don't see such description in current doc

**Comment by aswaterman** [comment]
Ah, I misread it.  The new text is misleading, as it suggests the register index can only go up to 31.  Could you rephrase it, or just delete the clause "giving access to elements up to element 31".

**Comment by wwang0** [comment]
OK.  will come up with new PR
